module/support/logging/DataLogging/data/config/DataLogging.yaml

```diff
@@ -10,26 +10,26 @@ output:
 messages:
   # Input
   message.platform.RawSensors: true
-  message.input.Image: true
-  message.output.CompressedImage: true
+  message.input.Image: false
+  message.output.CompressedImage: false
   # Vision
-  message.vision.Balls: true
+  message.vision.Balls: false
   message.vision.Goals: true
-  message.vision.GreenHorizon: true
+  message.vision.GreenHorizon: false
   message.vision.FieldLines: true
-  message.vision.VisualMesh: true
-  message.vision.BoundingBoxes: true
+  message.vision.FieldIntersections: true
+  message.vision.VisualMesh: false
+  message.vision.BoundingBoxes: false
   # Odometry and Localisation
-  message.localisation.Ball: true
-  message.localisation.Field: true
-  message.localisation.Robot: true
-  message.input.Sensors: true
-  message.support.FieldDescription: true
+  message.localisation.Ball: false
+  message.localisation.Field: false
+  message.localisation.Robot: false
+  message.input.Sensors: false
+  message.support.FieldDescription: false
   # Behaviour state
-  message.behaviour.state.WalkState: true
-  message.behaviour.state.Stability: true
+  message.behaviour.state.WalkState: false
   # Logging and Support
-  message.support.nusight.Overview: true
-  message.eye.DataPoint: true
-  message.nuclear.LogMessage: true
-  message.nuclear.ReactionStatistics: true
+  message.support.nusight.Overview: false
+  message.eye.DataPoint: false
+  message.nuclear.LogMessage: false
+  message.nuclear.ReactionStatistics: false
```

Comment:
this looks like you've made a specific datalogging set up for the optimisation routine - I'd recommend instead reverting this and making a role-specific config for the optimisation.